packages/fxa-auth-server/fxa-oauth-server/scripts/prepare-new-signing-key.js

```diff
@@ -1,49 +0,0 @@
-#!/usr/bin/env node
-
-/* This Source Code Form is subject to the terms of the Mozilla Public
- * License, v. 2.0. If a copy of the MPL was not distributed with this
- * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
-
-/*
```

Comment:
I believe @jrgm and @jbuck use prepare-new-signing-key, retire-old-signing-key, and sops-key-config to manage the OIDC and JWT access token signing keys, we'll need those still.

Reply:
in `fxa-auth-server/scripts`